plugins/org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -284,12 +284,10 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 							}
 							
 							// outline API
-							fragment = adapter.FragmentAdapter {
-								fragment = outline.OutlineTreeProviderFragment auto-inject {}
-							}
-							fragment = adapter.FragmentAdapter {
-								fragment = outline.QuickOutlineFragment auto-inject {}
-							}
+							fragment = ui.outline.OutlineTreeProviderFragment2 {}
```

Comment:
Currently we do not inject anything into that particular fragment, but if we create a project for clients, we want to to work as long and without surprises as possible. Thus if a client changes e.g. generateStubs on the level of the workflow, it should be applied to all fragments that have a property with that name. Thus auto-inject helps on new projects in general, but may be unnecessary for Xtend in particular.